src/include/connection.h

```diff
@@ -560,7 +560,6 @@ struct __wt_connection_impl {
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
 #define WT_JSON_OUTPUT_ERROR 0x1u
 #define WT_JSON_OUTPUT_MESSAGE 0x2u
-#define WT_JSON_OUTPUT_PROGRESS 0x4u
     /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
```

Comment:
Shouldn't this line be updated too after running `s_all`?

Reply:
Hmm, I don't follow. After removing `WT_JSON_OUTPUT_PROGRESS`, `s_all` isn't updating anything further here.

Reply:
Sorry my bad, I don't why I thought this should be updated by `s_all`.